wolox-books/src/store.js

```diff
@@ -1,15 +1,20 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
+import { getListOfBooks } from '@/services/bookService'
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
-    loginStatus: false
+    loginStatus: false,
+    booksList: []
```

Comment:
Since we now have 2 clearly very different variables, I think it's time to make different modules for each. Check them out https://vuex.vuejs.org/guide/modules.html